counsel.el

```diff
@@ -2986,6 +2986,38 @@ include attachments of other Org buffers."
             :action 'counsel-locate-action-dired
             :caller 'counsel-org-file))
 
+(defvar org-entities)
+(defvar org-entities-user)
+
+;;;###autoload
+(defun ivy-insert-org-entity ()
```

Comment:
This should probably read `counsel-org-insert-entity` or `counsel-org-entity` and have its own `;;**` heading like the other surrounding `counsel-org-*` commands.